rollup.config.js

```diff
@@ -32,7 +32,7 @@ export default {
   ],
   watch: {
     clearScreen: false
-  }
+  },
```

Comment:
can you rever this?

Reply:
What exactly should I revert here? Wasn't it already there before? Haha sorry can't really remember